src/test/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindCommandTest.java

```diff
@@ -159,5 +159,51 @@ public void invalidOptionsPagingStateWithSort() throws Exception {
           .extracting(ConstraintViolation::getMessage)
           .contains("pagingState is not supported with sort clause");
     }
+
+    @Test
```

Comment:
In addition to this, there should probably be an integration test to verify limit is enforced?